demos/common/cpp/pipelines/src/async_pipeline.cpp

```diff
@@ -69,11 +69,9 @@ int64_t AsyncPipeline::submitData(const InputData& inputData, const std::shared_
 
     auto internalModelData = model->preprocess(inputData, request);
 
-    request->SetCompletionCallback([this,
-        frameID,
-        request,
-        internalModelData,
-        metaData] {
+    request->SetCompletionCallback(
+        [this, frameID, request, internalModelData, metaData]
+        {
```

Comment:
```suggestion
        [this, frameID, request, internalModelData, metaData] {
```
as we agreed before, opening bracket should be at the end of the expression line